tcms/rpc/tests/test_testcasestatus.py

```diff
@@ -0,0 +1,57 @@
+from tcms.rpc.tests.utils import APITestCase
+from tcms.testcases.models import TestCaseStatus
+from tcms.tests.factories import (
+    ProductFactory,
+    TestCaseFactory,
+    TestPlanFactory,
+    UserFactory,
+    VersionFactory,
+)
+
```

Comment:
Missing coverage for the permissions_required decorator, see APIPermissionsTestCase and rpc/tests/test_testexecution.py for hints.